src/stories/Components/Sidebar/Sidebar.mdx

```diff
@@ -1,12 +1,18 @@
 import { Meta, Story, Canvas, Controls } from '@storybook/blocks'
 import * as SidebarStories from './Sidebar.stories'
-
+import { createSourceCodeLink } from '../../helpers/sourceCodeLink.js'
 import { createAlert } from '../Alert/Alert.js'
 
 <Meta of={SidebarStories} />
 
 # Sidebar
 
+<div
+  dangerouslySetInnerHTML={{
```

Comment:
@Jeremy-Walton This is quite the property value name... Assuming this isn't "dangerous" in our use case.

Reply:
It is not. It's an annoying React thing as there is no other way to interpolate content into the page. We are actually already doing this for the alerts we show on a few pages.

The only reason it is "dangerous" is if you are injecting user input or malformed content into the page as raw HTML. In our case this is preprocesses and doesn't show in the end. We control what is being "injected" so it can't cause issues.